src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -44,7 +44,7 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
             Log.Trace($"Creating pull request sourceBranch:{sourceBranch} targetBranch:{targetBranch} description:{description}");
             return new GitPullRequest
             {
-                Title = $"{Options.InsertionName} Insertion Into {Options.VisualStudioBranchName} ",
+                Title = $"{Options.InsertionName} '{Options.BranchName}/{Options.SpecificBuild}' Insertion into {Options.VisualStudioBranchName} ",
```

Comment:
You'll want to pass through and consume `buildToInsert` from [here](https://github.com/dotnet/roslyn-tools/blob/e22a64f07f5c84b261ce57c10ef29f2757d97a79/src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs#L110) (and also line 100 above) (or `buildVersion`).  `Options.SpecificBuild` is a command-line override that will only be used in manual cases.